CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is `Visual Studio` changing the file encoding?

Reply:
Yes, it did. I reverted them with vs code.